src/pages/home/sidebar/SignInButton.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable rulesdir/onyx-props-must-have-default */
+import React from 'react';
+import {View} from 'react-native';
+import PressableWithoutFeedback from '../../../components/Pressable/PressableWithoutFeedback';
+import Button from '../../../components/Button';
+import styles from '../../../styles/styles';
+import * as Session from '../../../libs/actions/Session';
+import useLocalize from '../../../hooks/useLocalize';
+import CONST from '../../../CONST';
+
+function SignInButton() {
+    const {translate} = useLocalize();
+
+    return (
+        <PressableWithoutFeedback
+            accessibilityLabel={translate('sidebarScreen.buttonMySettings')}
+            accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
+            onPress={Session.signOutAndRedirectToSignIn}
```

Comment:
This caused https://github.com/Expensify/App/issues/34764 since `signOutAndRedirectToSignIn` got the first param as `true` always. It should have been `false` instead.